### PR TITLE
en_ending_hero OK

### PR DESCRIPTION
**Pull request by fullgrowngaming**

### CODE REVIEW

File: src/overlays/actors/ovl_En_Ending_Hero/z_en_ending_hero.c [hunk 1]
```diff
@@ -21,16 +23,73 @@ const ActorInit En_Ending_Hero_InitVars = {
...
+            this->unk242 = 0;
+            this->unk240 = (s16)randZeroOneScaled(60.0f) + 0x14;
+        }
+    }
+    this->actionFunc(this, globalCtx);
+    Actor_SetVelocityAndMoveYRotationAndGravity(&this->actor);
+    func_800B78B8(globalCtx, &this->actor, 20.0f, 20.0f, 50.0f, 0x1D);
+}
+
+UNK_PTR D_80C1E970[] = { 0x06007350, 0x06009590, 0x06009F90, 0x0600A790, 0x0600AB90 };
```
Comment (Kenix3) [inline src/overlays/actors/ovl_En_Ending_Hero/z_en_ending_hero.c:70]: These look to be textures that come from the object file.
We don't currently have types for textures, but you should at least make a symbol for each of these segmented pointers if it's still OK.

File: src/overlays/actors/ovl_En_Ending_Hero/z_en_ending_hero.c [hunk 2]
```diff
@@ -21,16 +23,73 @@ const ActorInit En_Ending_Hero_InitVars = {
...
+UNK_PTR D_80C1E984[] = { 0x06007750, 0x0600A390, 0x0600A490 };
```
Comment (Kenix3) [inline src/overlays/actors/ovl_En_Ending_Hero/z_en_ending_hero.c:71]: These look to be textures that come from the object file.
We don't currently have types for textures, but you should at least make a symbol for each of these segmented pointers if it's still OK.

**Review by Kenix3 — CHANGES_REQUESTED** [review]

File: src/overlays/actors/ovl_En_Ending_Hero/z_en_ending_hero.c (hunk 2 above)
Comment (fullgrowngaming) [reply src/overlays/actors/ovl_En_Ending_Hero/z_en_ending_hero.c]: I converted them to symbols, and also went ahead and moved the arrays into the function and made them static

**Review by Rozelette — APPROVED** [review]
LGTM

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**